### PR TITLE
Improve way to write class resources with try-with-resource

### PR DESCRIPTION
**Pull request by renanmachad**
Based in this discussion <a href="https://github.com/quarkusio/quarkus/discussions/41704">here</a> i open this PR to improve a small piece of code 👍🏻

### CODE REVIEW

File: core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
     @Override
     public Iterable<? extends Completion> getCompletions(Element element, AnnotationMirror annotation, ExecutableElement member,
-            String userText) {
+                                                         String userText) {
```
Comment (mkouba) [inline core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java:127]: This does not look like the formatter plugin was used, or?

**Comment by mkouba** [comment]
Out of curiosity, what does the `:Art` prefix in the PR description mean?

I guess that @geoand read it as Arc and therefore requested a review from me... :smile:

**Comment by geoand** [comment]
Pretty much :)

File: core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java (hunk 1 above)
Comment (renanmachad) [reply core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java:127]: yes i think i make a mistake using `CTRL + ALT + L` on Intelij.
i need revert this?

**Comment by renanmachad** [comment]
> Out of curiosity, what does the `:Art` prefix in the PR description mean?
> 
> I guess that @geoand read it as Arc and therefore requested a review from me... 😄

I'm used to it making commits with some prefixes, like :fix , :bug, but i forgot that i making a PR on Quarkus and not on my work

**Comment by mkouba** [comment]
> > Out of curiosity, what does the `:Art` prefix in the PR description mean?
> > I guess that @geoand read it as Arc and therefore requested a review from me... 😄
> 
> I'm used to it making commits with some prefixes, like :fix , :bug, but i forgot that i making a PR on Quarkus and not on my work

I see. And what exactly does `:Art` mean?

**Comment by renanmachad** [comment]
> > > Out of curiosity, what does the `:Art` prefix in the PR description mean?
> > > I guess that @geoand read it as Arc and therefore requested a review from me... 😄
> > 
> > 
> > I'm used to it making commits with some prefixes, like :fix , :bug, but i forgot that i making a PR on Quarkus and not on my work
> 
> I see. And what exactly does `:Art` mean?

Refactor of some piece of code to improve him, like more clear, more beautiful.
<a href="https://gitmoji.dev/">reference</a>

**Comment by renanmachad** [comment]
> > > > Out of curiosity, what does the `:Art` prefix in the PR description mean?
> > > > I guess that @geoand read it as Arc and therefore requested a review from me... 😄
> > > 
> > > 
> > > I'm used to it making commits with some prefixes, like :fix , :bug, but i forgot that i making a PR on Quarkus and not on my work
> > 
> > 
> > I see. And what exactly does `:Art` mean?
> 
> Refactor of some piece of code to improve him, like more clear, more beautiful. [reference](https://gitmoji.dev/)

@mkouba  is something wrong? You laugh make me insecure about if i make a mistake or is okay

**Comment by mkouba** [comment]
> > > > > Out of curiosity, what does the `:Art` prefix in the PR description mean?
> > > > > I guess that @geoand read it as Arc and therefore requested a review from me... 😄
> > > > 
> > > > 
> > > > I'm used to it making commits with some prefixes, like :fix , :bug, but i forgot that i making a PR on Quarkus and not on my work
> > > 
> > > 
> > > I see. And what exactly does `:Art` mean?
> > 
> > 
> > Refactor of some piece of code to improve him, like more clear, more beautiful. [reference](https://gitmoji.dev/)
> 
> @mkouba is something wrong? You laugh make me insecure about if i make a mistake or is okay

@renanmachad  Nonono, it's fine. I'm not used to these gitmojis and the fact that `:art:` means `Improve structure / format of the code.` made me laugh...

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**